archive/android_bootldr.ksy

```diff
@@ -0,0 +1,36 @@
+meta:
+  id: android_bootldr
+  title: Qualcomm Snapdragon (MSM) bootloader.img format
+  file-extension: img
+  tags:
+    - archive
+    - android
+  license: CC0-1.0
+  endian: le
+doc-ref: https://android.googlesource.com/device/lge/hammerhead/+/7618a7/releasetools.py
+seq:
+  - id: magic
+    contents: BOOTLDR!
+  - id: images
+    -orig-id: num_images
+    type: u4
```

Comment:
Actually, `num_images` would be a far better name - you can't tell that `images` is only a number of images just from the name (I'd expect an attribute called `images` to have type `image[]`, i.e. an array of objects of type `image`). This also corresponds to recommended conventions in the [KSY Style Guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id):

> One SHOULD use the following rules to maintain consistency across various KSY files. Doing that would maintain the "principle of least surprise" and make life easier to end-users, reducing amount of guesswork.
> 
> * For simple non-repeated fields, use a simple singular form — e.g. `width`, `header`, `transaction_id`, file.
> * For an array of objects (i.e. with `repeat: something`), use plural form — e.g. `files`, `transactions`.
> * (...)
> * For fields that designate **number / count** of something (in particular, number of repetitions of some other structure), use `num_` prefix and plural form — i.e. `num_questions`, `num_blocks`, `num_nodes`

```suggestion
  - id: num_images
    type: u4
```

Also don't forget about

https://github.com/kaitai-io/kaitai_struct_formats/blob/d2500791ffe1b3d8b6af58552d3ddb275e2beb1b/archive/android_bootldr.ksy#L23-L26

while renaming (or just do `sed -i 's/\<images\>/num_images/g' <ksy-file>`).